llvm/tools/llvm-readobj/ELFDumper.cpp

```diff
@@ -7574,30 +7573,38 @@ template <class ELFT> void LLVMELFDumper<ELFT>::printBBAddrMaps() {
       continue;
     }
     for (const BBAddrMap &AM : *BBAddrMapOrErr) {
-      DictScope D(W, "Function");
-      W.printHex("At", AM.Addr);
+      DictScope FD(W, "Function");
+      if (AM.BBRanges.empty())
```

Comment:
Are there scenarios where BBRanges would be completely empty? If so, what are those scenarios?

Reply:
Not really. Except for tests. I will add one.